examples/proxy-server/src/main/resources/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+    <title>
+      Armeria animation
+    </title>
+  </meta>
+</head>
+<style>
+
+.ascii-art {
+  display: flex;
+  justify-content: center;
+  font-family: Courier, monospace;
```

Comment:
How about using a nicer font? We could source one from Google Fonts. Our official web site uses Inconsolata, so it'd be nice if we use the same font or something similar.

Reply:
I thought Inconsolata didn't work but I checked it again and it's working. I might do something wrong. :(